test/system_SUITE.erl

```diff
@@ -0,0 +1,446 @@
+-module(system_SUITE).
+-compile([export_all]).
+
+-include_lib("common_test/include/ct.hrl").
+-include_lib("amqp_client/include/amqp_client.hrl").
+
+-define(SERVER_REJECT_CLIENT, {tls_alert,"unknown ca"}).
+
+all() ->
+    [
+     library,
+     invasive_SSL_option_change,
+     validation_success_for_AMQP_client,
+     validation_failure_for_AMQP_client,
+     whitelisted_certificate_accepted_from_AMQP_client_regardless_of_validation_to_root,
+     removed_certificate_denied_from_AMQP_client,
+     installed_certificate_accepted_from_AMQP_client,
+     whitelist_directory_DELTA,
+     replaced_whitelisted_certificate_should_be_accepted,
+     ensure_configuration_using_binary_strings_is_handled
+    ].
+
+suite() ->
+    [{timetrap, {seconds, 25}}].
```

Comment:
25 seconds is a bit short for my aging laptop :-) I suspect it won't be enough for our Jenkins VMs or Travis.

Can we increase this to one minute?